src/index.html

```diff
@@ -29,24 +30,26 @@
           <h2>octocat</h2>
 
           <ul>
-            <li>GitHub</li>
-            <li>San Francisco</li>
-            <li>email link</li>
-            <li>website</li>
-            <li>Joined on Jan 25, 2011</li>
+            <li id="company"><span class="octicon octicon-organization"></span>  GitHub</li>
+            <li id="location"><span class="octicon octicon-location"></span>  San Francisco</li>
+            <li id="email"><span class="octicon octicon-mail"></span>  email link</li>
+            <li id="blog"><span class="octicon octicon-link"></span>  website</li>
+            <li id="created_at"><span class="octicon octicon-clock"></span>  Joined on Jan 25, 2011</li>
           </ul>
 
-          <span class="numbers">
-            <h3>1.1k</h3>
-            <h3>2</h3>
-            <h3>6</h3>
-          </span>
-
-          <span class="category">
-            <h5>Followers</h5>
-            <h5>Starred</h5>
-            <h5>Following</h5>
-          </span>
+          <table>
+            <tr class="numbers">
+              <td>1.1k</td>
+              <td>2</td>
+              <td>6</td>
+            </tr>
+            <tr class="category">
+              <th>Followers</th>
+              <th>Starred</th>
+              <th>Following</th>
+            </tr>
+          </table>
```

Comment:
`<table>`...? Gross. This looks more like three grid columns _inside of a grid column_...! ZOMG!